tutorials/multi/2_node_classification.py

```diff
@@ -0,0 +1,257 @@
+"""
+Single Machine Multi-GPU Minibatch Node Classification
+======================================================
+
+In this tutorial, you will learn how to use multiple GPUs in training a
+graph neural network (GNN) for graph classification.
+
+(Time estimate: 8 minutes)
+
+This tutorial assumes that you have read the :doc:`Training GNN with Neighbor
+Sampling for Node Classification <../large/L1_large_node_classification>`
+tutorial. It also assumes that you know the basics of training general
+models with multi-GPU with ``DistributedDataParallel``.
+
+.. note::
+
+   See :doc:`the multi-GPU graph classification
+   tutorial <1_graph_classification>`
+   for an overview of using ``DistributedDataParallel`` with DGL.
```

Comment:
Can we tell readers which specific paragraphs/section they will need to read?

Reply:
Done.